src/database/DataTypes/TemperatureBreach.js

```diff
@@ -93,7 +93,11 @@ TemperatureBreach.schema = {
     startTimestamp: { type: 'date', optional: true },
     endTimestamp: { type: 'date', optional: true },
     location: { type: 'Location', optional: true },
-    temperatureBreachConfiguration: { type: 'TemperatureBreachConfiguration', optional: false },
+    thresholdMaxTemperature: { type: 'double' },
+    thresholdMinTemperature: { type: 'double' },
+    thresholdDuration: { type: 'double' },
+    acknowledged: { type: 'boolean', default: false },
```

Comment:
```suggestion
    acknowledged: { type: 'bool', default: false },
```

Was getting this in the metro bundler but it went away with the above: 
<img width="707" alt="Screen Shot 2020-12-22 at 11 58 55 AM" src="https://user-images.githubusercontent.com/65875762/102830390-18d9d900-444e-11eb-94b4-4b922fc8dac5.png">

Reply:
b1a87e9

woooooooops